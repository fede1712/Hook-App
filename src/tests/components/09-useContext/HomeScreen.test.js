import React from 'react'
import { mount } from 'enzyme'

import { HomeScreen } from "../../../components/09-useContext/HomeScreen"
import { UserContext } from '../../../components/09-useContext/UserContext'
import { user } from '../../fixtures/demoTodos'

describe('Tests in <HomeScreen />', () => {


    const wrapper = mount(
        <UserContext.Provider value={{ user }}>
            <HomeScreen />
        </UserContext.Provider>
    )

    test('should render <HomeScreen /> correctly', () => {

        expect(wrapper).toMatchSnapshot()


    })


})
