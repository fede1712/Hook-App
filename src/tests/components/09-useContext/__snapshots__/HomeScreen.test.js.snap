// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HomeScreen /> should render <HomeScreen /> correctly 1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "fede",
   "email": "fede@fede.com"
}
  </pre>
</div>
`;
